pkg/controller/networkpolicy/tier.go

```diff
@@ -143,6 +151,25 @@ func (n *NetworkPolicyController) InitializeTiers() {
 	}
 }
 
+// waitForAPI waits for the Antrea API /readyz endpoint to report "ok"
+func (n *NetworkPolicyController) waitForAPI() error {
+	req, err := http.NewRequest("GET", "/readyz", nil)
```

Comment:
This seems quite wrong. Where is this request supposed to go? There is no host, no port. I tested this with a simple Go program and it doesn't seem to work. Can you check the error returned by `client.Do`, I don't think it's nil?